### PR TITLE
Jj starting on UI

### PR DESCRIPTION
**Pull request by jessjno**
used bootstrap to add navbar and CDN assets with a dropdown resources option, added the required sign in and alerts and updated sample data

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -6,11 +6,27 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+    <%= render partial: "shared/cdn_assets" %>
```
Comment (aldoportillo) [inline app/views/layouts/application.html.erb:9]: Nice good formatting

File: app/views/shared/_navbar.html.erb [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<nav class="navbar navbar-expand-lg bg-light" mb-4>
+  <div class="container-fluid">
+    <!--<a class="navbar-brand" href="#">Navbar</a>-->
```
Comment (aldoportillo) [inline app/views/shared/_navbar.html.erb:3]: you can delete this if you aren't using it

**Review by aldoportillo — APPROVED** [review]

**Review by Testell — APPROVED** [review]
looks great

**Review by tluu5 — APPROVED** [review]
You good to go.

**Pull request merged**